web/modules/weather_routes/weather_routes.routing.yml

```diff
@@ -48,3 +48,12 @@ weather_routes.alert-info:
   requirements:
     # Public endpoint
     _access: 'TRUE'
+
+weather_routes.afd:
+  path: '/afd'
+  defaults:
+    _controller: \Drupal\weather_routes\Controller\AFDController::content
+  options:
+    no_cache: 'TRUE'
```

Comment:
:shakes_fist:

It'd be dandy if Drupal included query parameters in the cache key, but it appears that it caches the route, not the URL. Alas!